meson.build

```diff
@@ -127,18 +156,21 @@ foreach t : ['', '-static']
   pkgconf.set('prefix', join_paths(get_option('prefix')))
   pkgconf.set('libdir', '${prefix}/@0@'.format(get_option('libdir')))
   pkgconf.set('VERSION', meson.project_version())
+  pkglibs = cpp_lib
+  if libm_dep.found()
+    pkglibs += ' -lm'
+  endif
+  if pthread_dep.found()
+    pkglibs += ' -lpthread'
+  endif
   if t == '-static'
-    do_install = false
-    pkgconf.set('LIBS', '-lstdc++ -lpthread -lm')
```

Comment:
why stdc++ is not be specified？

Reply:
iOS and macOS do not have stdc++ anymore and was removed from the toolchain a couple of years ago.  They use libc++.  Android has also deprecated their stdc++ module in favor of libc++ and would be removed in a future NDK release.

Reply:
Also, stc++ is specified by default through the cpp_lib variable ans is only overridden for android, macos and ios